src/libs/Navigation/AppNavigator/BaseDrawerNavigator.js

```diff
@@ -27,33 +28,43 @@ const propTypes = {
 };
 const Drawer = createDrawerNavigator();
 
-const BaseDrawerNavigator = props => (
-    <Drawer.Navigator
-        defaultStatus={props.isSmallScreenWidth ? 'open' : 'closed'}
-        sceneContainerStyle={styles.navigationSceneContainer}
-        drawerContent={props.drawerContent}
-        screenOptions={{
-            cardStyle: styles.navigationScreenCardStyle,
-            headerShown: false,
-            drawerType: getNavigationDrawerType(props.isSmallScreenWidth),
-            drawerStyle: getNavigationDrawerStyle(
-                props.windowWidth,
-                props.windowHeight,
-                props.isSmallScreenWidth,
-            ),
-            swipeEdgeWidth: 500,
-        }}
-    >
-        {_.map(props.screens, screen => (
-            <Drawer.Screen
-                key={screen.name}
-                name={screen.name}
-                component={screen.component}
-                initialParams={screen.initialParams}
-            />
-        ))}
-    </Drawer.Navigator>
-);
+const BaseDrawerNavigator = (props) => {
+    const content = (
+        <Drawer.Navigator
+            defaultStatus={props.isSmallScreenWidth ? 'open' : 'closed'}
+            sceneContainerStyle={styles.navigationSceneContainer}
+            drawerContent={props.drawerContent}
+            screenOptions={{
+                cardStyle: styles.navigationScreenCardStyle,
+                headerShown: false,
+                drawerType: getNavigationDrawerType(props.isSmallScreenWidth),
+                drawerStyle: getNavigationDrawerStyle(
+                    props.isSmallScreenWidth,
+                ),
+                swipeEdgeWidth: 500,
+            }}
+        >
+            {_.map(props.screens, screen => (
+                <Drawer.Screen
+                    key={screen.name}
+                    name={screen.name}
+                    component={screen.component}
+                    initialParams={screen.initialParams}
+                />
+            ))}
+        </Drawer.Navigator>
+    );
+
+    if (!props.isSmallScreenWidth) {
+        return (
+            <View style={styles.navigationSceneFullScreenWrapper}>
```

Comment:
That's what I tried first, and I found that the sceneContainerStyle targets a different UI component. 

Adding the View wrapper was necessary because it was the only way to wrap the Drawer and the Screen contents and apply the rounded corners to them.

Reply:
Got it. Does having the extra `View` around everything else cause issues as well? Something like

```js
<View style={props.isSmallScreenWidth ? styles.navigationSceneFullScreenWrapper : undefined}>
     ... other stuff
```

Reply:
Also no worries if there is no way to keep all the JSX together. I think it's just easier to parse without the `content` variable.